Update parameters for legacy Exchange tokens (#12360)
MicrosoftDocs/office-docs-powershell: exchange/exchange-ps/exchange/Set-AuthenticationPolicy.md

In Exchange 2019, this example re-enables Basic authentication for Exchange Reporting Web Services in the authentication policy named Research and Development Group.

### Example 3
```powershell

> Set-AuthenticationPolicy -Identity "Legacy Exchange Tokens" -BlockLegacyExchangeTokens

Reviewer: ```suggestion
Set-AuthenticationPolicy -Identity "LegacyExchangeTokens" -BlockLegacyExchangeTokens
```

```

> In Exchange Online, this example blocks legacy Exchange tokens from being issued to Outlook add-ins. Since this switch applies to the entire organization, the authentication policy specified with the Identity parameter is ignored.

Reviewer: ```suggestion
In Exchange Online, this example blocks legacy Exchange tokens from being issued to Outlook add-ins. The switch applies to the entire organization, and the Identity parameter must be set to the value "LegacyExchangeTokens". Specific authentication polices can't be applied.
```

### -AllowLegacyExchangeTokens
This parameter is available only in the cloud-based service.

> The AllowLegacyExchangeTokens switch specifies whether to allow legacy Exchange tokens for Outlook add-ins. You don't need to specify a value with this switch.

Reviewer: ```suggestion
The AllowLegacyExchangeTokens switch specifies to allow legacy Exchange tokens to be issued to Outlook add-ins. You don't need to specify a value with this switch.
```

Legacy Exchange tokens include Exchange user identity and callback tokens.

> This switch applies to the entire organization. Although the Identity parameter is required, its value is ignored. You can pass any non-empty value as the Identity parameter.

Reviewer: ```suggestion
The switch applies to the entire organization. The Identity parameter is required and must be set to the value "LegacyExchangeTokens". Specific authentication polices can't be applied.
```

### -BlockLegacyExchangeTokens
This parameter is available only in the cloud-based service.

> The BlockLegacyExchangeTokens switch specifies whether to block legacy Exchange tokens for Outlook add-ins. You don't need to specify a value with this switch.

Reviewer: ```suggestion
The BlockLegacyExchangeTokens switch specifies to block legacy Exchange tokens being issued to Outlook add-ins. You don't need to specify a value with this switch.
```

Legacy Exchange tokens include Exchange user identity and callback tokens.

> This switch applies to the entire organization. Although the Identity parameter is required, its value is ignored. You can pass any non-empty value as the Identity parameter.

Reviewer: ```suggestion
The switch applies to the entire organization. The Identity parameter is required and must be set to the value "LegacyExchangeTokens". Specific authentication polices can't be applied.
```
